chain/client/src/client_actor.rs

```diff
@@ -1151,9 +1141,9 @@ impl ClientActorInner {
 
         // We try to produce block for multiple heights (up to the highest height for which we've seen 2/3 of approvals).
         if latest_known.height + 1 <= self.client.doomslug.get_largest_height_crossing_threshold() {
-            debug!(target: "client", "Considering blocks for production between {} and {} ", latest_known.height + 1, self.client.doomslug.get_largest_height_crossing_threshold());
+            tracing::debug!(target: "client", height_start = latest_known.height + 1, height_end = self.client.doomslug.get_largest_height_crossing_threshold(), "considering blocks for production");
```

Comment:
Consider assigning the `height_start` and `height_end` to variables before the `if` condition above (which uses these same expressions in its condition already.)